features/DeviceController.feature

```diff
@@ -35,6 +35,9 @@ Feature: Device Manager device controller
       | collection | "devices"       |
     Then The document "tenant-kuzzle":"devices":"DummyTemp-detached" content match:
       | metadata.enrichedByBeforeUpdateDevice | true |
+    And I refresh the collection "tenant-kuzzle":"devices"
```

Comment:
You don't need to refresh the collection since you are not retrieving documents with search